docs/profiling.md

```diff
@@ -135,15 +147,18 @@ For profiling GPU work, you should use the tool corresponding to your GPU's vend
 
 Note that while RenderDoc is a great debugging tool, it is _not_ a profiler, and should not be used for this purpose.
 
-### Graphics work
+### Tracy RenderQueue
 
-Finally, a quick note on how GPU programming works. GPUs are essentially separate computers with their own compiler, scheduler, memory (for discrete GPUs), etc. You do not simply call functions to have the GPU perform work - instead, you communicate with them by sending data back and forth over the PCIe bus, via the GPU driver.
+While it doesn't provide as much detail as vendor-specific tooling, Tracy can also be used to coarsely measure GPU performance.
 
-Specifically, you record a list of tasks (commands) for the GPU to perform into a CommandBuffer, and then submit that on a Queue to the GPU. At some point in the future, the GPU will receive the commands and execute them.
+Add the `RenderDiagnosticsPlugin` to your app, and then compile with Bevy's `trace_tracy` feature. GPU spans will show up in a separate row at the top of Tracy, labeled as `RenderQueue`.
 
-In terms of where your app is spending time doing graphics work, it might manifest as a CPU bottleneck (extracting to the render world, wgpu resource tracking, recording commands to a CommandBuffer, or GPU driver code), or it might manifest as a GPU bottleneck (the GPU actually running your commands).
+> [!NOTE]
+> Due to dynamic clock speeds, GPU timings will have large frame-to-frame variance, unless you use an external tool to lock your GPU clocks to base speeds. When measuring GPU performance via Tracy, only look at the MTPC column of Tracy's statistics panel, and not individual frame data.
+<!-- markdownlint-disable MD028 -->
```

Comment:
I usually use medians rather than means. Medians are not as sensitive to outliers. Plus looking at the Tracy comparison histograms (the red vs yellow) gives a good feel. I do also look at the standard deviation but that’s usually visually apparent from the graph anyway.

Reply:
I don't actually know what MTPC is - is it the mean? 

Either way, agree, but I don't think tracy has any good way to check the median besides manually estimating from the histogram.

Regardless tools like nsight/rgp will be much better for stable and detailed measurements, this is meant to just be a quick aid to help people when reviewing PRs.

Reply:
I’m now not sure if it’s only in the comparison view with the histogram graphs or if it can also be seen elsewhere. But in the comparison view it calculates the median, mean, stddev and shows them so if they’d like standard spans they’ll be there :)

MTPC is mean time per call. Anyway, not blocking.